huaweicloud/services/waf/resource_huaweicloud_waf_domain.go

```diff
@@ -105,13 +105,12 @@ func ResourceWafDomainV1() *schema.Resource {
 				Default:  false,
 			},
 			"charging_mode": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				ForceNew:    true,
-				Default:     PaidType,
-				Description: "schema: Internal",
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  PaidType,
```

Comment:
```suggestion
				Default:  PaidTypePrePaid,
```

Reply:
As we discussed, the variable name has been used by other providers, so let's keep it as is.